HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs

```diff
@@ -29,12 +33,15 @@ namespace GraphicalEditor
     public partial class MainWindow : Window, INotifyPropertyChanged
     {
         private Canvas _canvas;
-
+        private MapObjectController mapObjectController;
+        private IMapObjectServices mapObjectServices;
+        private IMapObjectRepository mapObjectRepository;
         public List<MapObject> AllMapObjects { get; set; }
 
-        private IRepository repository;
+        private IRepository fileRepository;
```

Comment:
The old name is better (it might be a file repository right now, but not necessarily forever - which is why the interface exists).